yt/yt/server/node/exec_node/cache_location.cpp

```diff
@@ -206,7 +206,7 @@ bool TCacheLocation::ScheduleDisable(const TError& reason)
             WaitFor(RemoveChunks())
                 .ThrowOnError();
             WaitFor(BIND(&TCacheLocation::SynchronizeActions, MakeStrong(this))
-                .AsyncVia(GetAuxPoolInvoker())
+                .AsyncVia(GetAuxPoolInvoker(EWorkloadCategory::Idle, {})) // TODO what to pass here
```

Comment:
Is the idle descriptor used here now too?

Reply:
Yes, `EWorkloadCategory` conveniently implicitly converts into a `TWorkloadDescriptor`. I learned it the hard way and now use it